src/Api/Billing/Models/Requests/CreateClientOrganizationRequestBody.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Bit.Api.Utilities;
+using Bit.Core.Enums;
+
+namespace Bit.Api.Billing.Models.Requests;
+
+public class CreateClientOrganizationRequestBody
```

Comment:
A representation of the request sent by the client to create a new client organization. We need significantly less information than we do when spinning up a traditional organization.